drivers/mysql/mysql.go

```diff
@@ -69,7 +69,7 @@ func Open(connURL string) (drivers.Driver, error) {
 		return nil, &drivers.AppError{Driver: driverName, OrigErr: err, Message: "failed to sanitize url from custom parameters"}
 	}
 
-	driverConfig, err := mergeConfigWithParams(customParams, defaultConfig)
+	driverConfig, err := mergeConfigWithParams(customParams, *defaultConfig)
```

Comment:
Instead of derefencing this, I'd suggest to convert this to a function to return a struct pointer. So every call returns a new struct.